tests/selenium/webui-selenium-test.py

```diff
@@ -72,66 +72,128 @@ def setUp(self):
 
-        # CONFIRMATION:
-        # Clicks on 'submit'
+        # Submit restore
         self.wait_and_click(By.XPATH, '//input[@id="submit"]')
-        # Confirms alert that has text 'Are you sure ?'
+
+        # Confirms alert
         self.assertRegexpMatches(self.close_alert_and_get_its_text(), r'^Are you sure[\s\S]$')
-        # switch to dashboard to prevent that modals are open
+
+        # switch to dashboard to prevent that modals are open before logout
         self.wait_and_click(By.XPATH, '//a[contains(@href, "/dashboard/")]', By.XPATH, '//div[@id="modal-002"]//button[.="Close"]')
-        #self.assertRegexpMatches(self.close_alert_and_get_its_text(), r"^Oops, something went wrong, probably too many files.")
         self.close_alert_and_get_its_text()
 
-        # LOGOUT:
+        # Logout
+        self.logout()
+
+    def test_run_default_job(self):
+        self.login()
+        self.wait_and_click(By.ID, 'menu-topnavbar-job')
+        self.wait_and_click(By.LINK_TEXT, 'Run')
+        # Open the job list
+        self.wait_and_click(By.XPATH, '(//button[@data-id="job"])')
+        # Select the first job
+        self.wait_and_click(By.XPATH, '(//li[@data-original-index="1"])')
+        # Start it
+        self.wait_and_click(By.ID, 'submit')
+        self.wait_and_click(By.ID, 'menu-topnavbar-dashboard')
         self.logout()
 
+    def test_run_configured_job(self):
+
+        driver = self.driver
+        driver.get(self.base_url + '/')
+        self.login()
+        self.wait_and_click(By.ID, 'menu-topnavbar-job')
+        self.wait_and_click(By.LINK_TEXT, 'Run')
+
+        Select(driver.find_element_by_id('job')).select_by_visible_text('BackupCatalog')
+        Select(driver.find_element_by_id('client')).select_by_visible_text(self.client)
+        Select(driver.find_element_by_id('fileset')).select_by_visible_text('SelfTest')
+        Select(driver.find_element_by_id('storage')).select_by_visible_text('File')
+        Select(driver.find_element_by_id('pool')).select_by_visible_text('Incremental')
+        Select(driver.find_element_by_id('level')).select_by_visible_text('Incremental')
+        driver.find_element_by_id('priority').clear()
+        driver.find_element_by_id('priority').send_keys('5')
+
+        driver.find_element_by_css_selector("span.glyphicon.glyphicon-calendar").click()
+
+        self.wait_and_click(By.XPATH, '//td[3]/a/span')
+        self.wait_and_click(By.XPATH, '//td[3]/a/span')
+
+        self.wait_and_click(By.CSS_SELECTOR, 'span.input-group-addon')
+        self.wait_and_click(By.ID, 'submit')
+        self.wait_and_click(By.ID, 'menu-topnavbar-dashboard')
+
+        self.logout()
+
+    def test_rerun_job(self):
+        self.login()
+        self.wait_and_click(By.ID, "menu-topnavbar-client")
+        self.wait_and_click(By.LINK_TEXT, self.client)
+        # Select first backup in list
+        self.wait_and_click(By.XPATH, '//tr[@data-index="0"]/td[1]/a')
+        # Press on repeat button
```

Comment:
it is called "rerun" button

Reply:
Some of my comments have already been addressed with later changes. Please combine matching commits into a single commit  (git rebase -i).

Reply:
I rebased my commits and will get back to your comments tomorrow.